src/main/java/htsjdk/variant/variantcontext/Allele.java

```diff
@@ -200,6 +200,11 @@ protected Allele(final Allele allele, final boolean ignoreRefState) {
     public final static Allele NO_CALL = new Allele(NO_CALL_STRING, false);
     public final static Allele NON_REF_ALLELE = new Allele(NON_REF_STRING, false);
 
+    public final static Allele SV_SIMPLE_DEL = new Allele(createBracketedSymbAlleleString(StructuralVariantType.DEL), false);
```

Comment:
Can you just call `toSymbolicAltAllele()` on the enum values directly, and get rid of `createBracketedSymbAlleleString()`?